### PR TITLE
Capitalization, Product Names, and Wordings

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
### :chains: Related Resources

* https://chefio.atlassian.net/browse/CHEF-2997
* The PR https://github.com/chef/automate/pull/7948 will be deleted once this one is merged.
* https://chefio.atlassian.net/browse/CHEF-10980

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/elasticsearch.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
 +++
-title = "Elasticsearch"
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/elasticsearch.md:2]: Elasticsearch has a lowercase _s_

**Review by IanMadd — CHANGES_REQUESTED** [review]

File: components/docs-chef-io/content/automate/elasticsearch.md (hunk 1 above)
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/elasticsearch.md:2]: It has already been fixed.

File: components/docs-chef-io/content/automate/elasticsearch.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+title = "ElasticSearch"
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/elasticsearch.md:2]: This is not a lowercase _s_

**Review by IanMadd — CHANGES_REQUESTED** [review]
Look through here again. You still have "ElasticSearch" in a bunch of places.

File: components/docs-chef-io/content/automate/backup.md [hunk 3]
```diff
@@ -250,7 +250,7 @@ where `bucket_name` is the name of the GCS bucket and `base_path` is an optional
 
 ## Delete Backups
 
-To delete backups from a running instance of Chef Automate, first find the relevant backup ID with `chef-automate backup list` and then delete the backup using [`chef automate backup delete ID`]({{< relref "cli/#chef-automate-backup-delete" >}}).
+To delete backups from a running instance of Chef Automate, first find the relevant backup ID with `chef-automate backup list` and then delete the backup using [`Chef Automate backup delete ID`]({{< relref "cli/#chef-automate-backup-delete" >}}).
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/backup.md:253]: This is code and not regular text:

```suggestion
To delete backups from a running instance of Chef Automate, first find the relevant backup ID with `chef-automate backup list` and then delete the backup using [`chef-automate backup delete ID`]({{< relref "cli/#chef-automate-backup-delete" >}}).
```

**Review by IanMadd — APPROVED** [review]

**Pull request merged**